experimental/GModule/GModule.jl

```diff
@@ -536,7 +541,7 @@ end
 if isdefined(Hecke, :stub_composition_factors)
   function Hecke.stub_composition_factors(x::Vector{T}) where {T}
     F = base_ring(x[1])
-    h = Oscar.iso_oscar_gap(F)
+    h = Oscar.ring_iso_oscar_gap(F)
```

Comment:
We renamed Oscar.ring_iso_oscar_gap to Oscar.iso_oscar_gap ...

Reply:
just done, but still:
```
julia> Oscar.RepPc.reps(abelian_closure(QQ)[1], G)
ERROR: failed to convert GAP.GapObj to any known type:
 GAP: !Z(137)^0
Stacktrace:
  [1] gap_to_julia(x::GAP.GapObj; recursive::Bool)
    @ GAP ~/.julia/dev/GAP/src/gap_to_julia.jl:321
  [2] #gap_to_julia#29
    @ ~/.julia/dev/GAP/src/gap_to_julia.jl:94 [inlined]
  [3] #gap_to_julia#27
    @ ~/.julia/dev/GAP/src/gap_to_julia.jl:89 [inlined]
  [4] #36
    @ ~/.julia/dev/GAP/src/gap_to_julia.jl:204 [inlined]
```
I thought this was solved?